Testscripts/Linux/customKernelInstall.sh

```diff
@@ -379,6 +379,14 @@ function InstallKernel() {
         esac
         customKernelFilesUnExpanded="${CustomKernel#$LOCAL_FILE_PREFIX}"
         LogMsg "Removing packages that do not allow the kernel to be installed"
+        query_customKernel_install=$(rpm -qp --queryformat='%{Version}-%{Release}.%{Arch}\n' ${customKernelFilesUnExpanded})
```

Comment:
I don't understand this new logic. There is $isKernelUpgraded variable with new value in CommonFunctions library. Why does it need another evaluation for 'query_customKernel_install'?

Reply:
The new variable "query_customKernel_install" is added in bash script to get the kernel version from customKernel RPM file and check with the installed kernel version on VM. In commonfunctions, variable $isKernelUpgraded gives the state as true/false for upgrade. 
New logic implemented here is to sucessfully Capture the VHD,if customKernel passed is same as that of kernel version in the VM.

Reply:
Your logic eventually determines if the custom kernel is installed or not. This is not about what version of custom kernel is installed. Please use $isKernelUpgraded

Reply:
The main motive of PR is to continue VHD creation incase of given customKernel version and kernel version of VM is same.
We need to handle to update the status CUSTOM_KERNEL_ALREADY_INSTALLED/CUSTOM_SUCCESS to log file in shell script else the it returns false as it fails for 		line: 603 if ( $kernelSuccess -eq $jobCount ) condition.

Reply:
I don't understand. There are 2 changes in a PR. Are they relevant or not? Are you fixing one in the library and the other in the test script? If the change is in the library, can this variable use in the test script?